app.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const all_animals = [];
+
+function Animal (horn){
+   this.image_url = horn.image_url;
+   this.title = horn.title;
+   this.description = horn.description;
+   this.keyword = horn.keyword;
+   this.horns = horn.horns;
+ all_animals.push(this);
+}
+
+Animal.prototype.render = function(){

```

Comment:
Line 14 and 15 are a great way to do a "proof-of-life" check. Once you know that is working and you've moved on to actually instantiating the Animal objects from the json file you should remove these lines. As it is now they will create an extra section on the index page that holds blank data (because the Animal object is blank), and this is no longer a useful check.